index.js

```diff
@@ -22,9 +40,29 @@ function readCaches(readers, key, options) {
   return promiseUntil(firstFound, readers, [key, options]);
 }
 
-function writeCaches(writers, key, value, options) {
-  const never = () => false;
-  return promiseUntil(never, writers, [key, value, options]);
+function writeCaches(writers, key, cacheValue, options) {
```

Comment:
I didn't think that promiseUntil added any value here. An iterative solution seems fine and gives us more control over where to stop.